azure-pipelines.yml

```diff
@@ -121,6 +121,16 @@ jobs:
         NUMPY: '1.19'
         CONDA_ENV: travisci
         TEST_START_INDEX: 17
+      py39_np115:
+        PYTHON: '3.9'
+        NUMPY: '1.15'
```

Comment:
There is no Python 3.9 build of NumPy 1.15.  Will need to bump this to 1.16.

Reply:
done, thanks